src/components/AssignmentTexter/Controls.jsx

```diff
@@ -509,8 +511,8 @@ export class AssignmentTexterContactControls extends React.Component {
 
     const otherResponsesLink =
       currentInteractionStep &&
-        currentInteractionStep.question.filteredAnswerOptions.length > 6 &&
-        filteredCannedResponses.length ? (
+      currentInteractionStep.question.filteredAnswerOptions.length > 6 &&
```

Comment:
From here down in this file all the changes are whitespace changes because my editor autoformatted the code.